index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Redressed&display=swap" rel="stylesheet">
+    <title>Document</title>
```

Comment:
- [ ] Write the name of your website instead of the word inside `<title>` tag

Reply:
- [ ]  Hello Elyor! I think it is preferable to `kindly` suggest to the student that they have the places they need to change, rather than ordering them as politely as possible.

Example: 
> It would be much better to let know users about your website name by writing the name of your website inside of the `<title>` tag. For more info: [HTML <title> Tag](https://www.w3schools.com/tags/tag_title.asp)

```suggestion
    <title>Your website name</title>
```

Reply:
- [ ]  Hello Elyor! I think it is preferable to `kindly` suggest to the student that they have the places they need to change and some suggestions, rather than ordering them as `kindly` as possible.

Example: 
> It would be much better to let know users about your website name by writing the name of your website inside of the `<title>` tag. For more info: [HTML <title> Tag](https://www.w3schools.com/tags/tag_title.asp)

```suggestion
    <title>Your website name</title>
```